code/controllers/subsystems/ticker.dm

```diff
@@ -438,6 +438,23 @@ var/datum/controller/subsystem/ticker/SSticker
 
 	to_world("<B><span class='notice'>Welcome to the pre-game lobby!</span></B>")
 	to_world("Please, setup your character and select ready. Game will start in [pregame_timeleft] seconds.")
+
+	// Compute and, if available, print the ghost roles in the pre-round lobby. Begone, people who do not ready up to see what ghost roles will be available!
```

Comment:
Does this comment mean you want this to only show up to people that have readied?

Reply:
No, the original suggestion was to do this to reduce the people who do not ready up, that is a reference to it